contrib/epee/include/net/abstract_tcp_server2.h

```diff
@@ -86,6 +90,181 @@ namespace net_utils
     public i_service_endpoint,
     public connection_basic
   {
+  private:
+    using string_t = std::string;
```

Comment:
This "alias everything" style breaks from existing coding conventions, and it would preferable to revert to something closer to the existing style.

Reply:
+1 on this. Things like `std::string` or `std::mutex` shouldn't be renamed at all to avoid confusion. Long `epee` and `boost` names can probably be renamed.